js/lib/bands.js

```diff
@@ -170,7 +170,7 @@ function bandPlot(bandDivId, bandPathTextBoxId, dataFilePaths, dosFile, showFerm
                 $("#" + bandDivId + "bt-togglePdos").addClass("button-white");
                 $("#" + bandDivId + "bt-togglePdos").removeClass("button");
 
-                for (var i = 1; i < theBandPlot.dosSeries.length; i++) {
+                for (var i = 2; i < theBandPlot.dosSeries.length; i++) {
```

Comment:
What is the index for? Can you add a comment for the local variable here?

Reply:
This is the index to toggle visibility of the PDOS. Since one more dataset is added at the head. We need to start from 2. 1 is the total DOS.

Reply:
thanks for the explanation, can you add a comment so the developers who take over the maintenance duty in the future can easily understand it?

Reply:
Yes, I add comments in the code.